### ISSUE TITLE
Extra `{id: ...}` in filter queries? 

### ISSUE THREAD

**Issue by thought2** [panter/ra-data-prisma#24]
```
{
  "operationName": "actors",
  "variables": {
    "skip": 0,
    "take": 10,
    "orderBy": { "id": "asc" },
    "where": { "AND": [{}, { "publish": { "id": { "equals": true } } }] }
  },
  "query": "query actors($where: ActorWhereInput, $orderBy: ActorOrderByInput, $take: Int, $skip: Int) {\n  items: actors(where: $where, orderBy: $orderBy, take: $take, skip: $skip) {\n    id\n    firstName\n    lastName\n    picture\n    text\n    projects {\n      id\n      __typename\n    }\n    publish\n    __typename\n  }\n  total: actorsCount(where: $where)\n}\n"
}
```

this is the query sent by some filter set that looks like:

```
const ActorFilter = (props) => (
  <Filter {...props}>
    <SearchInput source="q" alwaysOn />
    <BooleanInput source="publish" />
  </Filter>
);
```

I'm getting a graph ql error when using the "publish" filter. I think according to the generated CRUD in the backend, it should be something like:

```
"where": { "AND": [{}, { "publish": { "equals": true } }] }
```

**Comment by macrozone** [comment]
@thought2 thats indeed weird. maybe its a bug with boolean filter

**Comment by macrozone** [comment]
@thought2 should be fixed in @ra-data-prisma/dataprovider@2.2.1

it should get released soon, i'll check later

**Comment by thought2** [comment]
yes, it think too that it's boolean specific. Thanks for fixing this!

I have similar problem with a relation many filter, will open up a second issue for this.
